docs/getting_started/installation.md

```diff
@@ -0,0 +1,47 @@
+# Installation
+
+## Requirements
+
+QuestionPy is a framework developed in Python.
+Therefore, you need a compatible Python version installed.
+A list of compatible versions can be found [here](faq.md#what-python-version-can-i-use-with-questionpy)
+
+---
+
+## Install QuestionPy SDK
+
+To create your own question type, QuestionPy provides a Software Development Kit (SDK).
+To install the SDK, you have to clone the QuestionPy SDK development branch from Git.
+
+
+[//]: # (Sadly the combination of an enumerated list and code doesn't really work with mkdocs.)
+
+<ol>
+   <li>Make sure you have <a href="https://git-scm.com/">Git</a> installed before you continue.</li>
+   <li>Clone the development branch into a local directory:
+      <br>
+      <div style="margin-top:1em;">
+         <code style="padding: 1em">git clone https://github.com/questionpy-org/questionpy-sdk.git</code>
```

Comment:
Die ```shell -Umgebung hat den Vorteil, dass man den Befehl mit einem Klick kopieren kann.

Reply:
Macht sich leider nicht gut in einer Liste.

Reply:
Ich habe mich jetzt gegen die html Liste entschieden. Sieht vielleicht nicht so gut aus, macht die Bearbeitung aber bei weiterm einfacher. Außerdem klappt das:
> Die ```shell -Umgebung hat den Vorteil, dass man den Befehl mit einem Klick kopieren kann.